graphics/transform_struct.cpp

```diff
@@ -25,10 +25,11 @@
 
 namespace Graphics {
 
-void TransformStruct::init(Common::Point zoom, uint32 angle, Common::Point hotspot, bool alphaDisable, TSpriteBlendMode blendMode, uint32 rgbaMod, bool mirrorX, bool mirrorY, Common::Point offset) {
```

Comment:
This commit should be squashed into the preceeding commit, as adding a variable to the constructor is something you should do when you introduce it.